content/en/docs/components/pipelines/v2/quickstart.md

```diff
@@ -1,16 +1,34 @@
 +++
 title = "Quickstart"
-description = "Getting started with Kubeflow Pipelines"
+description = "Get started with Kubeflow Pipelines"
 weight = 3
                     
 +++
+
+{{% stable-status %}}
+
 <!-- TODO: add UI screenshots for final pipeline -->
-In this tutorial we will deploy a KFP standalone deployment into an existing Kubernetes cluster, create and run a simple pipeline using the KFP SDK, view the pipeline on the KFP Dashboard, then create a more involved machine learning pipeline that uses additional KFP features.
+This tutorial helps you get started with KFP by doing the following:
```

Comment:
Since it's going to be a "long way down" for the reader, I suggest reorganizing the first paragraph into a mini-TOC that outlines the agenda and gives the reader some idea of the coverage of the guide.